rmgpy/data/kinetics/library.py

```diff
@@ -230,8 +277,45 @@ def convertDuplicatesToMulti(self):
         
         
     def load(self, path, local_context=None, global_context=None):
-        Database.load(self, path, local_context, global_context)
+        # Clear any previously-loaded data
+        self.entries = OrderedDict()
+        self.top = []
+
+        # Set up global and local context
+        if global_context is None: global_context = {}
+        global_context['__builtins__'] = None
+        global_context['True'] = True
+        global_context['False'] = False
+        if local_context is None: local_context = {}
+        local_context['__builtins__'] = None
+        local_context['entry'] = self.loadEntry
+        #local_context['tree'] = self.__loadTree
+        local_context['name'] = self.name
+        local_context['solvent'] = self.solvent
+        local_context['shortDesc'] = self.shortDesc
+        local_context['longDesc'] = self.longDesc
+        # add in anything from the Class level dictionary.
+        for key, value in Database.local_context.iteritems():
+            local_context[key]=value
         
+        # Process the file
+        f = open(path, 'r')
+        try:
+            exec f in global_context, local_context
```

Comment:
Codacy is complaining because you added an `exec` statement here. It got flagged twice, once by Bandit and once by PyLint.